hathor/p2p/states/peer_id.py

```diff
@@ -89,7 +101,6 @@ async def handle_peer_id(self, payload: str) -> None:
         data = json_loads(payload)
 
         peer = Peer.create_from_json(data)
-        peer.validate()
```

Comment:
Why was this removed? Either way this will conflict with the latest refactor.

Reply:
Because I had added this validation to be done inside of `create_from_json`. It seems you did this too in the PR merged to master, so it still made sense to remove this while I was fixing conflicts.